src/main/java/uk/gov/hmcts/darts/event/service/handler/DarStartHandler.java

```diff
@@ -33,6 +34,7 @@ public DarStartHandler(RetrieveCoreObjectService retrieveCoreObjectService,
     @Override
     @Transactional
     public void handle(DartsEvent dartsEvent, EventHandlerEntity eventHandler) {
+        DataUtil.preProcess(dartsEvent);
```

Comment:
Do we only wish to preProcess specific "types" of event? 

If not, could we preProcess within `EventDispatcherImpl.receive()`, which is closer to the point of entry of events into the system? That would avoid the need to preProcess in each individual handler.

Reply:
Good spot,

I think we already preProcess on both the Dispatcher service and the controller so I removed the redundant checks :)